doc/api/stream.md

```diff
@@ -1244,13 +1244,15 @@ changes:
     description: The `chunk` argument can now be a `Uint8Array` instance.
 -->
 
-* `chunk` {Buffer|Uint8Array|string|any} Chunk of data to unshift onto the
+* `chunk` {Buffer|Uint8Array|string|null|any} Chunk of data to unshift onto the
   read queue. For streams not operating in object mode, `chunk` must be a
   string, `Buffer` or `Uint8Array`. For object mode streams, `chunk` may be
   any JavaScript value other than `null`.
 * `encoding` {string} Encoding of string chunks. Must be a valid
   `Buffer` encoding, such as `'utf8'` or `'ascii'`.
 
+Passing `chunk` as `null` signals the end of the stream (EOF), after which no more data can be written.
```

Comment:
This will (I hope) cause the linter to fail because it is longer than 80 characters. Word wrap at 80 chars? To run the markdown linter, you can run `make lint-md` (or, if on Windows, `vcbuild lint-md`).